ethpm/package.py

```diff
@@ -128,6 +131,18 @@ def from_ipfs(cls, ipfs_uri: str) -> 'Package':
 
         return cls(package_data)
 
+    @classmethod
+    def from_registry(cls, registry_uri: str, w3: Web3) -> 'Package':
```

Comment:
Don't we have a URI type in `eth-typing`?

Reply:
not yet in a released version - but it's there in an approved pr